### ISSUE TITLE
Manual installation

### ISSUE THREAD

**Issue by hirejohnsalcedo** [software-mansion/react-native-reanimated#217]
Helps if we can an alternative installation method to `react-native link react-native-reanimated`. Most of the installation process is idiomatic, however I needed to find out what to import in `MainApplication.java` which is `import com.swmansion.reanimated.ReanimatedPackage;`

If this gets approval, I can open a PR

**Comment by osdnk** [comment]
#me2

**Comment by peymannaderi3** [comment]
react-native link react-native-reanimated doesn't work properly. the only way is installing manually!!

**Comment by ahilles107** [comment]
I also have error with `import com.swmansion.reanimated.ReanimatedPackage`. How it can be installed manually so react/android will see this package?

**Comment by matthewricecampbell** [comment]
@ahilles107 did you happen to figure out a solution? 
In my MainApplication.java `import com.swmansion.reanimated.ReanimatedPackage;` is showing as an unused import statement. And `new ReanimatedPackage()` is showing as Cannot resolve symbol.

**Comment by IronTony** [comment]
Any news about that?

**Comment by SourceCipher** [comment]
We need the manual installation for this library!

When do people realize that `react-native link` works only 1 out of 1000 times and it should be never even considered to be an only linking process! 

Without manual linking its impossible to install this library. I have searched all possible solutions on google and as soon as I added this library to my project, everything crashed, nothing compiles anymore no matter what I changed.
